tests/uc_oss/test_uc_oss_integration.py

```diff
@@ -0,0 +1,161 @@
+            # Take the first row of the training dataset as the model input example.
+            input_example = X.iloc[[0]]
+            # Log the model and register it as a new version in UC.
+            mlflow.sklearn.log_model(
+                clf,
+                "model",
+                # The signature is automatically inferred from the input example and
+                # its predicted output.
+                input_example=input_example,
+                registered_model_name=model_name,
+            )
+
+    try:
+        build_model()
+    except Exception as e:
+        assert False, f"Unhandled exception while attempting to build a model: {e}"
+
+    model_version = 1
+    model_uri = f"models:/{model_name}/{model_version}"
+    rm_desc = "UC-OSS/MLflow Iris model"
+    mv_desc = "Version 1 of the UC-OSS/MLflow Iris model"
+
+    # Load the model and do some batch inference.
+    try:
+        # By specifying the UC OSS model uri, mlflow will make UC OSS
+        # REST API calls to retrieve the model
+        loaded_model = mlflow.pyfunc.load_model(model_uri)
+        predictions = loaded_model.predict(X_test)
+        iris_feature_names = datasets.load_iris().feature_names
+        result = pd.DataFrame(X_test, columns=iris_feature_names)
+        result["actual_class"] = y_test
+        result["predicted_class"] = predictions
+        assert result[:4] is not None
+    except Exception as e:
+        assert False, f"Unhandled exception while loading a model: {e}"
+
+    try:
+        # list_artifacts will use the UC OSS model URI and make REST API calls to
+        # UC OSS to:
+        #   1) retrieve credentials (none for file based UC OSS)
+        #   2) use the storage location returned from UC OSS for the model version
+        #      list the artifacts stored in the location
+        mlflow.artifacts.list_artifacts(model_uri)
+    except Exception as e:
+        assert False, f"Unhandled exception while listing artifacts: {e}"
+
+    path = os.path.join("/tmp", "models", model_name, str(model_version))
+
+    try:
+        # download_artifacts will use the UC OSS model URI and make REST API calls
+        # to UC OSS to:
+        #   1) retrieve credentials (none for file based UC OSS)
+        #   2) copy the artifact files from the storage location to the
+        #      destination path
+        mlflow.artifacts.download_artifacts(
+            artifact_uri=f"models:/{model_name}/{model_version}",
+            dst_path=path,
+        )
+        requirements_path = f"{path}/requirements.txt"
+        assert os.path.exists(requirements_path), f"File {requirements_path} does not exist."
+        with open(requirements_path) as file:
+            lines = file.readlines()
+        assert len(lines) > 0
+    except Exception as e:
+        assert False, f"Unhandled exception while testing download_artifacts: {e}"
+
+    # Test get RM/MV works
+    model1 = mlflow.MlflowClient().get_registered_model(model_name)
```

Comment:
```suggestion
    client = mlflow.MlflowClient()
    model1 = client.get_registered_model(model_name)
```

let's reuse the client.